site/ko/blog/new-in-devtools-93/index.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: "layouts/blog-post.njk"
+title: "DevTools 업데이트 소식 (Chrome 93)"
+authors:
+  - jecelynyeen
+date: 2021-07-28
+updated: 2021-07-28
+description:
+  "수정 가능한 CSS 컨테이너 쿼리, 웹 번들 프리뷰, Console 에서 더 나은 문자열 제어 및 추가 업데이트 사항"
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/cNXHu1JnHCfRSnIr1bV7.jpg'
```

Comment:
I updated the hero image to Korean. Is that make sense because the "업데이트 소식" I put before the word "DevTools". https://developer-chrome-com.imgix.net/image/dPDCek3EhZgLQPGtEG3y0fTn4v82/cNXHu1JnHCfRSnIr1bV7.jpg?auto=format&w=1600